src/blitter/32bpp_sse_func.hpp

```diff
@@ -10,10 +10,16 @@
 #ifndef BLITTER_32BPP_SSE_FUNC_HPP
 #define BLITTER_32BPP_SSE_FUNC_HPP
 
+/* ATTENTION
+ * This file is compiled multiple times with different defines for SSE_VERSION and MARGIN_NORMAL_THRESHOLD.
+ * Be careful when declaring things with external linkage.
+ * Use internal linkage instead, i.e. "static".
+ */
+
```

Comment:
This is too far away to be in the diff when someone would remove the static. In the commit that introduced the bug I didn't look beyond the diff, so I would not have noticed this message and neither would the reviewer.

Reply:
What's your suggestion? Duplicate in front of every single function?

Reply:
I can also rename the file to ``.inl`` or something.

Reply:
Maybe something short like `/* Do not remove 'static', see attention above. */` just before each of the `static`s?

Reply:
I uses another define to obfuscate ``static``, so search/replace won't find it.